inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -47,33 +47,34 @@ IRParser::IRParser(size_t version, const std::vector<InferenceEngine::IExtension
     }
 }
 
-std::map<uint64_t, uint64_t> V10Parser::XmlDeserializer::map_type_in_function(const pugi::xml_node& node,
-    const std::string map_type) {
-    std::map<uint64_t, uint64_t> type_id_in_function;
-    uint64_t map_type_number = 0;
+V10Parser::XmlDeserializer::IoMap V10Parser::XmlDeserializer::updated_io_map(const pugi::xml_node& node) {
     auto body_node = node.child("body");
 
     if (body_node.empty()) {
         THROW_IE_EXCEPTION << "Missing body part.";
     }
-
     // Fill map: parameter/result id to parameter/result number in Function
+
+    auto extend_io_map = io_map;
+
     FOREACH_CHILD(layer, body_node.child("layers"), "layer") {
         auto type = XMLParseUtils::GetStrAttr(layer, "type");
 
-        if (type == map_type) {
+        if (type == "Parameter") {
+            auto id = XMLParseUtils::GetUIntAttr(layer, "id");
+            extend_io_map[io::INPUTS].insert({id, -1});
+        } else if (type == "Result") {
             auto id = XMLParseUtils::GetUIntAttr(layer, "id");
-            type_id_in_function.emplace(id, map_type_number);
-            map_type_number++;
+            extend_io_map[io::INPUTS].insert({id, -1});
```

Comment:
Should it be `io::OUTPUTS` or I am wrong?

Reply:
once again copy paste issue. Thanks